waku/v2/protocol/filter/server.go

```diff
@@ -69,6 +69,10 @@ func (wf *WakuFilterFullNode) SetHost(h host.Host) {
 	wf.h = h
 }
 
+func (wf *WakuFilterFullNode) Host() host.Host {
```

Comment:
The `Host` function in here, lightnode and relay aren't necessary as you would have access to the host via the WakuNode.

Reply:
Understood, done.